src/client/webpack.config.js

```diff
@@ -43,6 +43,9 @@ const webpackConfig = {
     ]),
     new webpack.optimize.CommonsChunkPlugin(/* chunkName= */'vendor', /* filename= */'vendor.js', function(module){
       return module.resource && module.resource.indexOf('node_modules') !== -1;
+    }),
+    new webpack.DefinePlugin({
```

Comment:
I had a look at seeing if you could write a config module on the fly and registered it in the bundle during build. Doesn't seem to be supported out of the box. You'd have to write a plugin I believe. 

For example 

```
// webpack.config.js
var config = {
  someSetting:'foo'
}
// where plugins are registered
new DymanicModulePlugin('my-dynamic-config', config);
```
```
// consumer.js
import config from 'my-dynamic-config';
console.log(config.someSetting);
```

Shame, but until such exists I guess the options are:
1. global variables are the only option. 
2. write a custom config object to disk at build time and configure a dynamic alias to it.

Point 1 (as per this PR) makes the most sense.